test/python/tf_unittest_runner.py

```diff
@@ -39,89 +42,145 @@ def main():
     optional.add_argument(
         '--run_test',
         help=
-        "Runs the testcase and returns the output. Eg:math_ops_test.math_ops_test.DivNoNanTest.testBasic"
+        "Runs the testcase and returns the output. Eg:math_ops_test.DivNoNanTest.testBasic"
     )
     parser._action_groups.append(optional)
     arguments = parser.parse_args()
 
-    all_dirs_to_path(arguments.tensorflow_path)
-
     if (arguments.list_tests):
-        list_tests(arguments.list_tests)
-
+        test_list = helper(arguments.tensorflow_path, arguments.list_tests)
+        print('\n'.join(test_list))
     if (arguments.run_test):
-        run_test(arguments.run_test)
+        test_list = helper(arguments.tensorflow_path, arguments.run_test)
+        run_test(test_list)
+
+
+def helper(tf_path, test_regex):
+    accepted_formats = [
+        "math_ops_test", "mat_ops_test.DivNoNanTest",
+        "math_ops_test.DivNoNanTest.testBasic", "math_ops_test.DivNoNanTest.*",
+        "math_ops_test.D*", "math_ops_test.*", "math_*_test", "math_*_*_test",
+        "math*_test"
+    ]
+    try:
+        module_list = regex_walk(tf_path, test_regex)
+    except:
+        module_list = []
+        print(
+            "Invalid module name.Use bazel query to get list of tensorflow python test modules"
```

Comment:
`name.Use` -> `name. Use`

`Use bazel query` ... maybe we can print the exact command, if its easy enough to print (i dont know the command, so not sure if its a small or a large thing)